### ISSUE TITLE
Repeated Notification Emails

### ISSUE THREAD

**Issue by shawnim** [input-output-hk/cardano-rt-view#153]
I am using cardano-rt-view 0.3.0 with cardano-node 1.25.1.
I received many repeated emails for what appears to be the same warning notification.
I received over 20 emails at intervals of about 2 minutes before I disabled notifications.
The emails were similar to:

Cardano RTView Notification
2021-02-06 01:11:57.961714913 UTC, from the node 'producer1': 3 slots were missed!

and

Cardano RTView Notification
2021-02-06 01:09:56.525397752 UTC, from the node 'producer1': 3 slots were missed!

Each email had an increasing timestamp.
I believe this is a problem because the Blockchain tab shows:

Missed slots number        3

I would expect it to only send 1 email.

**Comment by denisshevchenko** [comment]
> I received over 20 emails at intervals of about 2 minutes

Yes, by default RTView checks errors every 120 seconds and sends an email with all of them. But you can tune this period: https://github.com/input-output-hk/cardano-rt-view/blob/master/doc/technical-details/email-notifications.md#check-period
